### PR TITLE
[17.0][MIG] l10n_fi_edicode: Migration to 17.0

### PR DESCRIPTION
**Pull request by anikeenko-viktor**
Based on https://github.com/OCA/l10n-finland/pull/55

### CODE REVIEW

File: l10n_fi_edicode/models/res_partner_operator_einvoice.py [hunk 1]
```diff
@@ -32,22 +32,18 @@ class ResPartnerOperatorEinvoice(models.Model):
         ),
     ]
 
-    def name_get(self):
+    @api.depends("identifier")
+    def _compute_display_name(self):
         """
         Overwrite core method to add value of `identifier` ("Identifier") field
         into name of recors.
```
Comment (aisopuro) [inline l10n_fi_edicode/models/res_partner_operator_einvoice.py]: Since we're here, could we fix this typo as well? `recors` => `records`

**Review by aisopuro — COMMENTED** [review]
Minor improvement while we're here. Not critical to do if this is in a hurry.

File: l10n_fi_edicode/models/res_partner_operator_einvoice.py (hunk 1 above)
Comment (anikeenko-viktor) [reply l10n_fi_edicode/models/res_partner_operator_einvoice.py]: Fixed

**Comment by jarmokortetjarvi** [comment]
Looks good :+1:

**Comment by aisopuro** [comment]
@jarmokortetjarvi @mlaitinen since we have 2 reviews, I believe this can be merged without waiting for @hegenator (he's quite busy at the moment).

**Pull request merged**